etc/requirements_blaze.txt

```diff
@@ -1,3 +1,5 @@
 -e git://github.com/quantopian/datashape.git@bf06a41dc0908baf7c324aeacadba8820468ee78#egg=datashape-dev
 -e git://github.com/quantopian/odo.git@9e16310b5f2c3f05162145200db7e7908f0a866e#egg=odo-dev
 -e git://github.com/quantopian/blaze.git@7b8f70c15279c7ebacf2ac2f5f74acf08b300d71#egg=blaze-dev
+
+-e git://github.com/quantopian/empyrical.git@6c8d59c0af66bb1243277bdb2822e34480395bb0#egg=empyrical-master
```

Comment:
It's a temp commit so I could get travis to run. It's not going to live here.

See comment on 0980bfaf98e2a52a6de351d222891c9b9314e2c3